frontendGraphApi/src/main/java/com/virtuslab/gitmachete/frontend/graph/api/items/ICommitItem.java

```diff
@@ -18,4 +21,7 @@ default IBranchItem asBranchItem() {
   default ICommitItem asCommitItem() {
     return this;
   }
+
+  IGitMacheteCommit getCommit();
+  IGitMacheteNonRootBranch getContainingBranch();
```

Comment:
It can be misleading coz one commit can be on (be available form) many branches.
Maybe (I dunno) something like `getExclusivelyContainingBranch()` or something better that emphasize that this commit is available only from that branch

Reply:
Hmm that's a bit tricky here... coz still, there's never a guarantee that a given commit X will be available only from the one given branch A... it's just enough to create a new branch B at that commit X, and now X is available from both A&B. So `getExclusivelyContainingBranch` would be misleading as well :/

Reply:
It's actually "containing" in the machete sense, which is much more strict than "containing" in git sense.

branch A "contains" commit X in git sense => there's a path (possibly of zero length) from commit pointed by A to commit X

branch A "contains" commit X in git-machete sense => A is a non-root branch and X lies between commit pointed by A and A's fork point

Reply:
Not sure what name would best reflect that nuances... maybe I should just copy the above comment to javadoc? ;)

Reply:
Done ;)